src/App.tsx

```diff
@@ -191,17 +192,18 @@ export default function App() {
           return n.name;
         });
         setMeasureOptions(names);
-      });
-  }, []);
-
-  useEffect(() => {
-    fetch(`https://api.github.com/repos/cqframework/ecqm-content-r4/contents/bundles/measure`)
+        return fetch(`https://api.github.com/repos/cqframework/ecqm-content-r4/contents/bundles/measure`);
+      })
       .then(response => response.json())
```

Comment:
Ooops.
```suggestion
      .then(response => {
        if (response.status === 403) {
          if (response.headers.get('X-RateLimit-Reset') != null) {
            let resetTime = new Date(parseInt(response.headers.get("X-RateLimit-Reset") as string) * 1000);
            throw new Error(`GitHub Rate Limited until: ${resetTime}`);
          } else {
            throw new Error('Auth error with GitHub.')
          }
        }
        return response.json()
      })
```

Reply:
That shows this in the console.
![image](https://user-images.githubusercontent.com/11613067/104752647-eb1e4200-5724-11eb-9e5e-820858d9583b.png)

Reply:
sweeeeeet